Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -391,8 +359,21 @@ def bump_formula_pr
             #{user_message}
           EOS
         end
-        safe_system "hub", "pull-request", *hub_args, "-m", pr_message
-        safe_system "git", "checkout", *git_final_checkout_args, "-"
+        pr_title = "#{formula.name} #{new_formula_version}#{devel_message}"
+
+        begin
+          url = GitHub.create_pull_request(formula.tap.full_name, pr_title,
+                                     "#{username}:#{branch}", "master", pr_message)["html_url"]
+          if ARGV.include?("--no-browse")
+            puts url
+          else
+            exec_browser url
+          end
+        rescue GitHub::AuthenticationFailedError, GitHub::HTTPNotFoundError,
+               GitHub::RateLimitExceededError, GitHub::Error, JSON::ParserError
+          onoe "Unable to open pull request"
```

Comment:
Would be good to print out the exception here too. Also, given these are the same exceptions as before, may want to have a wrapper for handling these `rescues` or just a shared array of them all.

Reply:
Using a shared array now, updated the code & PR.

Reply:
My bad. Lost track of this PR. Took long to address this.